Services/MathJax/interfaces/interface.ilMathJaxConfigRespository.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ */
+interface ilMathJaxConfigRespository
+{
+    /**
+     * Get the MathJax Configuration
+     */
+    public function getConfig() : ilMathJaxConfig;
+
+    /**
+     * Update the MathJax Configuration
+     */
+    public function updateConfig(ilMathJaxConfig $config) : void;
+}
```

Comment:
An empty line is missing here. I suggest to apply the `PSR-2 + X` coding style by executing the `PHP-CS-FIXER` for all changed PHP files.

Reply:
OK, docs/development/code-style-configs/php-storm.xml seems to differ a bit from CI/PHP-CS-Fixer/code-format.php_cs regarding the newlines. Now I executed the fixer on all files.